terraform/modules/emr/templates/emr/poll_status_table.sh

```diff
@@ -44,7 +44,11 @@ log_message $MESSAGE "INFO" "NOT_SET" $PROCESS_ID "batch_emr" "poll_status_table
 
 count=0
 
-query="SELECT correlation_id FROM audit.data_pipeline_metadata_hive WHERE dataproduct = '${DATASOURCE}' AND upper(status) = 'COMPLETED' AND dateproductrun = '${EXPORT_DATE}';"
+if [[ -n "$5" ]]; then
+  SNAPSHOT_TYPE="$5"
+  query="SELECT correlation_id FROM audit.data_pipeline_metadata_hive WHERE dataproduct = '${DATASOURCE}' AND upper(status) = 'COMPLETED' AND dateproductrun = '${EXPORT_DATE}' AND snapshot_type = '${SNAPSHOT_TYPE}';"
+else query="SELECT correlation_id FROM audit.data_pipeline_metadata_hive WHERE dataproduct = '${DATASOURCE}' AND upper(status) = 'COMPLETED' AND dateproductrun = '${EXPORT_DATE}';"
```

Comment:
Has this change been tested in dev?

Reply:
Yes. I run the script inside the EMR

Reply:
Have you ran PT-1 or CBOL in dev though to ensure they still work?